skills_ml/algorithms/occupation_classifiers/classifiers.py

```diff
@@ -0,0 +1,112 @@
+import os
+import logging
+import json
+import tempfile
+
+from collections import Counter
+
+from gensim.models import Doc2Vec
+
+from skills_ml.algorithms.string_cleaners import NLPTransforms
+
+from skills_utils.s3 import download
+
+PATHTOMODEL = 'open-skills-private/model_cache/'
+LOOKUP = 'lookup_va_0605.json'
+
+MODEL_NAME = 'gensim_doc2vec_va_0605'
```

Comment:
I think we should document a bit about how this pretraining happened. What dataset, what time period, how many examples. Is there any of that metadata still easily available?